lib/modules/manager/composer/__snapshots__/extract.spec.ts.snap

```diff
@@ -16,201 +16,294 @@ exports[`modules/manager/composer/extract extractPackageFile() extracts dependen
       "datasource": "packagist",
       "depName": "ext-intl",
       "depType": "require",
+      "registryUrls": [
+        "https://packagist.org",
```

Comment:
are we sure we always want to set the default registry url?

Reply:
Maybe I need to rethink this 🤔